src/compass/utils/defaults/geo_cslc_s1.yaml

```diff
@@ -10,6 +10,10 @@ runconfig:
           safe_file_path:
           # Required. List of orbit (EOF) files (min=1)
           orbit_file_path:
+          # Required. Path to the burst data
+          burst_file_path:
```

Comment:
`burst_file_path` is the ASF file?

Reply:
It will be any gdal readable file that contains the data for one burst.